### PR TITLE
Add coverage, Fix for Lint

### LINKED ISSUE #27: Raise backend coverage
Backend coverage is dropping a lot. It is currently at ~50%.

Tests should be done at least for the models, api and repositories. Batch jobs may be more troublesome to test and can be deprioritized

### PR DESCRIPTION
**Pull request by ng95junwei**
Fixes #27

### CODE REVIEW

File: backend/src/repositories/ticker_covariance.py [hunk 1]
```diff
@@ -10,6 +11,7 @@ def get_covariance_matrix(symbols):
         cov_mat = np.ndarray(shape=(dim, dim))
         for i in range(dim):
             for j in range(i, dim):
+                print(symbols[i], symbols[j])
```
Comment (ricktjwong) [inline backend/src/repositories/ticker_covariance.py]: Will this flood the console too much?

Comment (ng95junwei) [reply backend/src/repositories/ticker_covariance.py]: its a booboo

File: backend/test/routes/test_allocation.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import unittest
+
+from models.abc import db
+from repositories import TickerCovarianceRepository, TickerRepository
+from server import server
+
+
+class TestTicker(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
```
Comment (ricktjwong) [inline backend/test/routes/test_allocation.py:11]: are these db methods usually camelCased?

**Review by ricktjwong — APPROVED** [review]
lgtm

File: backend/test/routes/test_allocation.py (hunk 2 above)
Comment (ng95junwei) [reply backend/test/routes/test_allocation.py:11]: This is from the default project so i assume its in the correct style

**Pull request merged**